src/index.html

```diff
@@ -11,7 +11,195 @@
+            min="1" 
+            max="100"
+            value="12"
+            name="age"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="birth"
+            placeholder="mm/dd/yyyy"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label>
+            I accept the term of the agreement
+          <input 
+            type="checkbox"
+            name="terms"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>
+          Registration
+        </legend>
+
+        <label class="form-field">
+          E-mail:
+          <input 
+            type="email" 
+            placeholder="email@example.com"
+            name="email"
+            required
+          >
+        </label>
+
+        <label>
+          Password:
+          <input 
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>
+          An interesting fact about you!
+        </legend>
+    
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input 
+            type="radio"
+            name="cat"
+            value="yes"
+            >
+            Yes 
+          </label>         
+          <label>
+            <input 
+            type="radio"
+            name="cat"
+            value="no"
+            >
+            No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input 
+            type="color"
+            name="color"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bad
+          <input 
+            type="time"
+            name="time"
+          >
+        </label>
+
+        <label class="form-field">
+          What are you favorite brand of cars
+          <select 
+            name="cars" 
+            multiple
+          >
+            <option value="BMW">
+              BMW
+            </option>
+            <option value="Audi">
+              Audi
+            </option>
+            <option value="Lada">
+              Lada
+            </option>
+          </select>
+        </label>
+
+        <label>
+          How do you rate our work?
+          <input 
+            type="range"
+            value="0"
+            min="0"
+            max="10"
+            name="range"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>
+          Additional info:
+        </legend>
+
+        <label class="form-field">
+          Comments:
+          <textarea 
+            name="feedback" 
+            cols="20" 
+            rows="2"
+          >
+          </textarea>
```

Comment:
```suggestion
          ></textarea>
```
to prevent this
<img width="304" alt="image" src="https://user-images.githubusercontent.com/53611910/205499295-32aa5080-52e1-45bf-aa20-dd81f229a24e.png">